### PR TITLE
explicitly state when an issue needs to be created

### PR DESCRIPTION
**Pull request by afeld**
This is a proposal. Feel free to suggest changes!

### CODE REVIEW

File: ProjectBoard.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 ## General Instructions
 
-### Radiating intent
-
-Let the rest of the team know you are starting an issue by to start an issue by copying the link of the issue into our private chat and @-ing the rest of the team.
+- An issue should be created when a task:
+  - Is the responsibility of our team (versus something like a personal HR-related task)
```
Comment (its-a-lisa) [inline ProjectBoard.md]: Should we be tracking our 'passion project' on the board as well?

Comment (its-a-lisa) [inline ProjectBoard.md]: Can we have a bullet specific to 'Action Item from other meeting'  -- I believe that is one of the last remaining aspects that might not be making it to the project board; thoughts?

Comment (afeld) [reply ProjectBoard.md]: If it does/would fall under the purview of the team, I think so. If it's "learn a musical instrument" (or the technology equivalent), probably not. That seem reasonable?

Comment (afeld) [reply ProjectBoard.md]: Done.

Comment (its-a-lisa) [reply ProjectBoard.md]: Created a passion project label at Monday's meeting

**Pull request merged**